app/services/dqt/client.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Dqt
```

Comment:
I wonder whether these files belong in `app/services` or `app/lib`. So far we've been putting miscellaneous "service" classes in `app/lib` so it might be better to be consistent with that, but I can see that these classes are more like proper "service" classes in that they interact with a service... so not sure!

Reply:
Yeah. They definitely don't belong where I've put them. I just stuck them all in the same directory while I was getting something working. I expect there'll be some services, some stuff in lib and a bunch of jobs when we build it properly.